File: prob5/prob5.py
from random import randrange

def rand_print(num):
	if num > 40:
		for x in range(40,num+1):
			if x % 2 != 0:
				print x
	else:
		for x in range(0,40+1):
			if x % 2 != 0:
				print x

rand_num = randrange(100)
print "The number is " + str(rand_num)
print ""
rand_print(rand_num)

File: prob05/prob5.js
var rand_print = function (rand_num){
	list = [];
	if (rand_num > 40) {
		for (var i = 40; i <= rand_num; i++) {
			if (i % 2 !== 0){
				list.push(i);
			}
		}
	} else {
		for (var i = 0; i < 40; i++) {
			if (i % 2 !== 0) {
				list.push(i);
			}
		}
	}
	return list;
}

var rand_num = (Math.floor((Math.random() * 100)))
document.write("The number is " + rand_num + "<br/>")
document.write(rand_print(rand_num))